Assignment8/Assignment8/MainWindow.xaml.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Threading;
+
+namespace Assignment8
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private readonly DispatcherTimer dispatcherTimer = new();
+
+        private DateTime _StartTime = DateTime.Now;
+        private string _Description;
+        private bool _IsStopped = true;
+        private int Entry = 1;
+
+        private void Description_TextChanged(object sender, TextChangedEventArgs e) => _Description = Description.Text;
+
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            dispatcherTimer.Tick += new EventHandler(DispatcherTimer_Tick);
```

Comment:
Event handler can be simplified.
```suggestion
            dispatcherTimer.Tick += DispatcherTimer_Tick;
```